taccsite_cms/static/site_cms/css/src/_imports/tools/x-drop-cap.css

```diff
@@ -1,19 +1,12 @@
 .x-drop-cap {
-  /* Drop the letter */
-  float: left;
   /* Estimate horizontal spacing to match magazine design */
   /* WARN: Firefox calculates `margin-…` differently here, so use `padding-…` */
   padding-right: 0.125em; /* approx: `1em` of normal text */
 
   font-size: 6em; /* approx: 3 lines, of normal text, tall */
 
-  /* (Firefox v73) Nudge letter up until it meets top of normal text */
-  /* CAVEAT: Also has effect in Safari and Chrome/Edge, but not enough */
-  padding-top: 0.0625em;
-  /* (Safari v13, Chrome/Edge v80) Nudge letter up more until it meets top of normal text */
-  /* MAGIC: Firefox does not need this */
-  /* SEE: https://bugzilla.mozilla.org/show_bug.cgi?id=1233271 */
-  /* MAGIC: Has no effect on Firefox */
-  /* SEE: https://bugzilla.mozilla.org/show_bug.cgi?id=290125 */
-  line-height: 0.65;
+  /* Not yet available on Firefox */
+  initial-letter: 3;
```

Comment:
**Uh oh.** This renders poorly on Firefox. Please use [Progressive Enhancement](https://developer.mozilla.org/en-US/docs/Glossary/Progressive_Enhancement) to do this this on browsers that support it, but not on browsers that don't. CSS offers [Feature Detection](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) via @supports.